webui/react/src/services/decoder.ts

```diff
@@ -58,6 +61,7 @@ export const mapV1GroupRole = (res: Sdk.V1GetRolesAssignedToGroupResponse): type
     id: role.roleId,
```

Comment:
do we still need `fromWorkspace`? 

also wondering if `scopeCluster` could be renamed here to be more clear/consistent... `fromCluster`? `fromGroup`?